weblate/static/loader-codemirror.js

```diff
@@ -61,6 +61,7 @@
       mode: "text/x-markdown",
       theme: "weblate",
       lineNumbers: false,
+      lineWrapping: true,
```

Comment:
```suggestion
      lineWrapping: true,
      viewportMargin: Infinity,
```

Needed to display more than 10 rows, see https://codemirror.net/demo/resize.html and https://codemirror.net/doc/manual.html#option_viewportMargin